growth-chart-app-modern/src/components/Spinner.tsx

```
@@ -0,0 +1,46 @@
+}
+
+const Spinner: React.FC<SpinnerProps> = ({
+  size = 'md',
+  color = 'text-blue-600 dark:text-blue-400',
+  className = ''
+}) => {
+  let sizeClasses = '';
+  switch (size) {
+    case 'sm': sizeClasses = 'h-6 w-6'; break;
+    case 'lg': sizeClasses = 'h-12 w-12'; break;
+    case 'md':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)